src/main/java/duke/TaskStorage.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+
+public class TaskStorage {
+    /**
+     * saveTask is used to store current tasks into a txt file
+     * @param prev_tasks File object to be written to
+     * @param tasks TaskList to be saved
+     */
+    public static void saveTask(File prev_tasks, TaskList tasks) {
```

Comment:
Perhaps saveTasks (similar to loadTasks above) or saveTaskList would be clearer in reflecting the methods usage?